### PR TITLE
Handling loading and disconnection 

### PR DESCRIPTION
**Pull request by Tom32i**

### CODE REVIEW

**Comment by Tom32i** [comment]
On connection:
capture d ecran 2015-03-15 a 21 20 10

On disconnection:
capture d ecran 2015-03-15 a 21 20 15

File: src/server/core/Server.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ Server.prototype.authorizationHandler = function(request, socket, head)
         return socket.end();
     }
 
+
     var sockect = new WebSocket(request, socket, head, ['websocket'], {ping: 5}),
```
Comment (JeremyJames) [inline src/server/core/Server.js]: sockect ?
Pourquoi le saut de ligne au dessus ?

File: src/client/service/Chat.js [hunk 2]
```diff
@@ -147,8 +148,22 @@ Chat.prototype.onTalk = function(e)
         player = this.room.getPlayerByClient(data.client),
         message = new Message(data.content, data.client, player ? player : {name: data.name, color: data.color}, data.creation);
```
Comment (emerick42) [inline src/client/service/Chat.js]: I'm sorry to revive old memories, but these `=` are not properly aligned :'(

File: src/server/core/Server.js (hunk 1 above)
Comment (Tom32i) [inline src/server/core/Server.js]: une erreur; à supprimer :)

**Comment by emerick42** [comment]
:trophy:

**Comment by JeremyJames** [comment]
:trophy:

**Pull request merged**